config/manifests/bases/jaeger-operator.clusterserviceversion.yaml

```
@@ -124,7 +124,7 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.31.0
+  replaces: jaeger-operator.v1.30.0
```

```suggestion
  replaces: jaeger-operator.v1.32.0
```
Does it not replace 1.32?